hazelcast/src/main/java/com/hazelcast/map/impl/eviction/EvictorImpl.java

```diff
@@ -151,10 +164,14 @@ protected int getEvictionStartIndex(long[] criterias, int evictableSize) {
     }
 
     @Override
-    public int findRemovalSize(RecordStore recordStore) {
+    public int findSizeToEvict(RecordStore recordStore) {
+        int currentPartitionSize = recordStore.size();
```

Comment:
Since a recordStore is always per partiton, what is the purpose if naming this variable currentPartitionSize? It is less informative than currenRecordStoreSize.